api/v1/storagecluster_types.go

```diff
@@ -184,11 +184,21 @@ type ManageCephBlockPools struct {
 	ReconcileStrategy    string `json:"reconcileStrategy,omitempty"`
 	DisableStorageClass  bool   `json:"disableStorageClass,omitempty"`
 	DisableSnapshotClass bool   `json:"disableSnapshotClass,omitempty"`
+	// StorageClassName specifies the name of the storage class created for ceph block pools
+	// +kubebuilder:validation:MaxLength=253
+	StorageClassName string `json:"storageClassName,omitempty"`
+	// KubeVirtStorageClassName specifies the name of the storage class created for ceph block pools
+	// for virtualization environment
+	// +kubebuilder:validation:MaxLength=253
+	KubeVirtStorageClassName string `json:"kubeVirtStorageClassName,omitempty"`
```

Comment:
Let's call it something like VirtualizationStorageClassName, As in the future we might create this storageclass for some other crd also apart from kubevirt.